packages/web/src/elements/dashboard.ts

```diff
@@ -26,13 +27,25 @@ export class MyDashboard extends SignalWatcher(LitElement) {
       <li>
         <span>${user.name}</span>
         <div class="user-actions">
-          <button @click=${() => selectUser(user)}>View Details ➡️</button>
-          <button @click=${() => deleteUser(user)} class="delete-btn">❌</button>
+          <button @click=${() => Actions.selectUser(user)}>View Details ➡️</button>
+          <button @click=${() => Actions.deleteUser(user)} class="delete-btn">❌</button>
         </div>
       </li>
     `;
   }
 
+  private renderCreateUserError() {
+    const error = state.error.value;
+    if (!error) return html``;
+    if (error.id === 'apiCreateUser') {
+      if (error.error._tag === 'parse-error') {
+        return html`<div class="error">Error creating user: the data was malformatted</div>`;
+      }
+      return html`<div class="error">Error creating user: there was an unknown error</div>`;
+    }
+    return html``;
+  }
```

Comment:
This shows that errors can be narrowed by function name AND by the _tag field on the error class